interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
Isn't a Node, here, just a datum in an RDF Statement? An IRI is not a _resource_, is my understanding.

Why do we call them bNodes, if those also aren't nodes?

[RDF Interfaces](http://www.w3.org/TR/rdf-interfaces/) uses NamedNode, and has many implementations. We already thought out much of this, why the desire to re-implement from scratch?